docs/planning.md

```diff
@@ -0,0 +1,81 @@
+feel free to use one of the provided [issue templates](https://github.com/eclipse-tractusx/sig-release/issues/new/choose) and describe your request.
+Please be aware, that not every feature request can be integrated and that we also cannot treat every issue with the highest priority.
+
+Every Release planning will be kicked off by two public alignment sessions. The dates and further details will be shared via
+[tractusx-dev](https://accounts.eclipse.org/mailing-list/tractusx-dev) mailing list.
+In addition to that, you can also find public meetings and info about how to join on our
+[Open Meetings](https://eclipse-tractusx.github.io/community/open-meetings) community page.
+Issues or bug reports, that should be discussed in these meetings, have to be opened prior to the meeting via
+our [issue templates](https://github.com/eclipse-tractusx/sig-release/issues/new/choose).
+
+### What can I expect
+
+We really welcome every contribution. Every Bug report and feature proposal takes time to prepare,
+is valuable to our project, and we very much appreciate this input.
+We are giving our best to give a first feedback in one week.
+If we should miss that, please stick with us and just use the commenting function to remind us of the issue.
+
+### Issue structure
+
+Our issues do have important properties, that enable our planning process. These are:
+
+- __Labels:__ We use them to indicate the involved teams (kit or foss component). A label for each involved component is added to an issue
+- __Issue Type:__ To separate between bugs, feature requests and release criteria, we use a custom field `Issue Type`
+- __Milestone:__ Every Tractus-X release is represented by a `Milestone`. You can use this field to get a rough idea about the ETA
+- __Status:__ The status field is used to integrate the progress of an issue
+
+### Issue statuses
+
+The following statuses are defined:
+
+- __Inbox:__ This is the initial status of all issues. It indicates, that involved components have to be identified and additional information gathered
+- __Backlog:__ If enough information is gathered, and we agreed to work on the issue, it is set from `Inbox` to `Backlog` to indicate it is ready for timeline planning
+- __Work in Progress:__ The issue is actively being worked on.
+- __Done:__ All relevant parts have been implemented and released
+
+### Issue process
+
+Every new feature proposal or bug report will be handled as issue in status `Inbox` initially. The alignment meetings are used to discuss the purpose and impact of the current issues.
+While in `Inbox` status, the involved components are discovered and respective `Labels` are added. If already possible, a desired `Milestone` can be set.
+Additionally, an `Assignee` is selected, who will coordinate efforts to solve the issue.
+
+After these details are clarified, an issue is moved to `Backlog` to open it for detailed timeline planning. In this status, discussions about a fitting `Iteration` is held.
```

Comment:
The fully refined status "backlog" is the entry point for our "open planning" ceremony.